src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -585,14 +585,13 @@ protected function getEnvParameters()
      */
     protected function buildContainer()
     {
-        foreach (array('cache' => $this->getCacheDir(), 'logs' => $this->getLogDir()) as $name => $dir) {
-            if (!is_dir($dir)) {
-                if (false === @mkdir($dir, 0777, true) && !is_dir($dir)) {
-                    throw new \RuntimeException(sprintf("Unable to create the %s directory (%s)\n", $name, $dir));
```

Comment:
Why is there a newline at the end? Seems pointless and should be removed in the new code IMO.